cernopendata_client/cli.py

```diff
@@ -284,14 +284,12 @@ def download_files(
             )
         )
         download_single_file(path=path, file_location=file_location)
-
-    if verify:
-        file_info_remote = get_file_info_remote(
-            recid, filtered_files=download_file_locations
-        )
-        file_info_local = get_file_info_local(recid)
-        click.secho("==> Verifying downloaded files for record {}.".format(recid))
-        verify_file_info(file_info_local, file_info_remote)
+        if verify:
+            file_info_remote = get_file_info_remote(
+                recid, filtered_files=[file_location]
+            )
+            file_info_local = get_file_info_local(recid)
+            verify_file_info(file_info_local, file_info_remote)
```

Comment:
Looks good in general :+1: 

```console
$ cernopendata-client download-files --recid5500 --filter-range 1-4 --verify
==> Downloading file 1 of 4
  -> File: ./5500/BuildFile.xml
  -> Progress: 0/0 kiB (0%)
==> Verifying file BuildFile.xml...
  -> expected size 305, found 92
ERROR: File size does not match.
```

However I noticed that when I use `--server` option, the verification does not seem to be done:

```console
$ cernopendata-client download-files --recid 5500 --filter-range 1-4 --verify --server http://localhost:5000
==> Downloading file 1 of 4
  -> File: ./5500/BuildFile.xml
  -> Progress: 7/7 kiB (100%)
==> Downloading file 2 of 4
  -> File: ./5500/HiggsDemoAnalyzer.cc
  -> Progress: 7/7 kiB (100%)
==> Downloading file 3 of 4
  -> File: ./5500/List_indexfile.txt
  -> Progress: 7/7 kiB (100%)
==> Downloading file 4 of 4
  -> File: ./5500/M4Lnormdatall.cc
  -> Progress: 7/7 kiB (100%)
==> Success!
```

Can you please have a look?

Reply:
Looking into it 👍

Reply:
This is OK:

```console
$ cernopendata-client download-files --recid 5500 --filter-range 1-4 --verify --server http://opendata.cern.ch
==> Downloading file 1 of 4
  -> File: ./5500/BuildFile.xml
  -> Progress: 0/0 kiB (100%)
==> Verifying file BuildFile.xml...
  -> expected size 305, found 305
  -> expected checksum adler32:ff63668a, found adler32:ff63668a
...
```

This is not:

```console
$ cernopendata-client download-files --recid 5500 --filter-range 1-4 --verify --server http://opendata-dev.cern.ch
==> Downloading file 1 of 4
  -> File: ./5500/BuildFile.xml
  -> Progress: 0/0 kiB (100%)
==> Downloading file 2 of 4
  -> File: ./5500/HiggsDemoAnalyzer.cc
  -> Progress: 81/81 kiB (100%)
...
```

Reply:
Done 👍